Doc/library/sqlite3.rst

```diff
@@ -25,6 +25,15 @@ The sqlite3 module was written by Gerhard Häring.  It provides an SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
+This document includes four main sections:
+
+* :ref:`sqlite3-tutorial` teaches how to use the sqlite3 module.
+* :ref:`sqlite3-reference` describes the classes and functions this module
+  defines.
+* :ref:`sqlite3-howtos` explains how to handle specific tasks.
```

Comment:
```suggestion
* :ref:`sqlite3-howtos` details how to handle specific tasks.
```

Avoid ambiguity/confusing in using "explain" for a non-explanation section (didn't catch this the first time, sorry)